src/common/motech-file-upload/motech-file-upload.html

```diff
@@ -1,5 +1,5 @@
 <div class="form-control input-auto input-large fileinput-exists" data-trigger="fileinput">
-    <span class="glyphicon glyphicon-file"></span>&nbsp;
+    <span class="icon-file"></span>&nbsp;
```

Comment:
Everything is alright. Just add this in 'base.icons.scss'. (It's working in my repo as I already have this property defined)
```
.icon-file{
    @include icon(file);
 }
```